tensorboard/data/experimental/experiment_from_dev_test.py

```diff
@@ -105,19 +101,29 @@ def stream_experiment_data(request, **kwargs):
                                 ]
                             ),
                         }
-                    )
-
-                    if pivot is None:  # Default behavior: pivot_table.
-                        pandas.testing.assert_frame_equal(
-                            dataframe,
-                            expected.pivot_table(
-                                ["value", "wall_time"], ["run", "step"], "tag"
-                            ),
-                            check_names=True,
-                        )
-                    else:  # pivot == False
+                        if include_wall_time:
+                            data["wall_time"] = np.concatenate(
+                                [
+                                    2.0 * np.arange(0, 10),
+                                    1.0 * np.arange(0, 10),
+                                    600.0 + 2.0 * np.arange(0, 10),
+                                    600.0 + np.arange(0, 10),
+                                ]
+                            )
+                        expected = pandas.DataFrame(data)
+                        if pivot is None:  # Default behavior: pivot_table.
+                            expected = expected.pivot_table(
+                                values=(
+                                    ["value", "wall_time"]
+                                    if include_wall_time
+                                    else "value"
+                                ),
+                                index=["run", "step"],
+                                columns="tag",
+                            ).reset_index()
+                            expected.columns.name = None
```

Comment:
I would suggest directly constructing the desired DataFrame here (i.e., with a direct call to `pandas.DataFrame`), rather than calling `pivot_table`, etc.

Reply:
Great suggestion. Done!